### PR TITLE
Operator Tensor Sum via IdentityOp

### PR DESCRIPTION
**Pull request by fzimmermann89**
Altrenative implemenation of Operator-Tensor Addition, using the existing IdentityOp and a new MultiIdentityOp (n tensors -> n tensors identiy)

### CODE REVIEW

File: src/mrpro/operators/IdentityOp.py [hunk 1]
```diff
@@ -42,3 +43,25 @@ def adjoint(self, x: torch.Tensor) -> tuple[torch.Tensor]:
             the input tensor
         """
         return (x,)
+
+
+class MultiIdentityOp(EndomorphOperator):
+    r"""The Identity Operator.
+
+    An endomorph Operator that returns multiple inputs unchanged.
```
Comment (ckolbPTB) [inline src/mrpro/operators/IdentityOp.py]: operator

File: src/mrpro/operators/IdentityOp.py [hunk 2]
```diff
@@ -42,3 +43,25 @@ def adjoint(self, x: torch.Tensor) -> tuple[torch.Tensor]:
...
+    """
+
+    @endomorph
+    def forward(self, *x: torch.Tensor) -> tuple[torch.Tensor, ...]:
+        """Identity of input.
```
Comment (ckolbPTB) [inline src/mrpro/operators/IdentityOp.py]: maybe "Identity applied to input"

File: src/mrpro/operators/IdentityOp.py [hunk 3]
```diff
@@ -42,3 +43,25 @@ def adjoint(self, x: torch.Tensor) -> tuple[torch.Tensor]:
             the input tensor
         """
         return (x,)
+
+
+class MultiIdentityOp(EndomorphOperator):
```
Comment (ckolbPTB) [inline src/mrpro/operators/IdentityOp.py]: Is this not covered in MultiIdentityOp.py?

File: tests/operators/test_operators.py [hunk 4]
```diff
@@ -134,6 +134,62 @@ def test_sum_operator_linearoperator():
...
+    a = DummyLinearOperator(rng.complex64_tensor((3, 3)))
+    b = rng.complex64_tensor((3,))
+    c = a + b
+    x = rng.complex64_tensor(3)
+    (y1,) = c(x)
+    y2 = a(x)[0] + b * x
+
+    torch.testing.assert_close(y1, y2)
+    assert isinstance(c, LinearOperator), 'LinearOperator + tensor should be a LinearOperator'
+    assert isinstance(c, Operator), 'LinearOperator + tensor should be an Operator'
```
Comment (ckolbPTB) [inline tests/operators/test_operators.py]: This of course does not hurt but it is not really provide additional information, does it?

File: src/mrpro/operators/IdentityOp.py (hunk 3 above)
Comment (fzimmermann89) [reply src/mrpro/operators/IdentityOp.py]: Ups. mistake in rebasing.

**Comment by fzimmermann89** [comment]
pre-commit.ci autofix

**Comment by fzimmermann89** [comment]
see #431

**Pull request closed without merging**